satpy/tests/reader_tests/test_sgli_l1b.py

```diff
@@ -0,0 +1,346 @@
+"""Tests for the SGLI L1B backend."""
+import sys
+from datetime import datetime, timedelta
+
+import dask
+import dask.array as da
+import h5py
+import numpy as np
+import pytest
+from dask.array.core import normalize_chunks
+from xarray import DataArray, Dataset, open_dataset
+
+from satpy.readers.sgli_l1b import HDF5SGLI
+
+START_TIME = datetime.now()
+END_TIME = START_TIME + timedelta(minutes=5)
+FULL_KM_ARRAY = np.arange(1955 * 1250, dtype=np.uint16).reshape((1955, 1250))
+MASK = 16383
+LON_LAT_ARRAY = np.arange(197 * 126, dtype=np.float32).reshape((197, 126))
+AZI_ARRAY = np.random.randint(-180 * 100, 180 * 100, size=(197, 126), dtype=np.int16)
+ZEN_ARRAY = np.random.randint(0, 180 * 100, size=(197, 126), dtype=np.int16)
+
+
+def test_open_dataset(sgli_vn_file):
+    """Test open_dataset function."""
+    from satpy.readers.sgli_l1b import SGLIBackend
+    res = open_dataset(sgli_vn_file, engine=SGLIBackend, chunks={})
+    assert isinstance(res, Dataset)
+    data_array = res["Lt_VN01"]
+    assert isinstance(data_array, DataArray)
+    assert isinstance(data_array.data, da.Array)
+    assert data_array.chunks == normalize_chunks((116, 157), data_array.shape)
+
+
+@pytest.fixture(scope="session")
```

Comment:
These fixtures could probably be set to "module" scope since they aren't used outside this module.

Reply:
Done